src/bootstrap-table.js

```diff
@@ -502,7 +505,11 @@
     formatRecordsPerPage (pageNumber) {
       return `${pageNumber} rows per page`
     },
-    formatShowingRows (pageFrom, pageTo, totalRows) {
+    formatShowingRows (pageFrom, pageTo, totalRows, totalNotFiltered) {
+      if(totalNotFiltered !== undefined && totalNotFiltered > 0 && totalNotFiltered < totalRows) {
```

Comment:
`eslint` problem, need to use `yarn lint --fix`

Reply:
done